build/yamls/antrea-windows.yml

```diff
@@ -126,6 +126,11 @@ data:
       # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
       # apiserver directly.
       #proxyAll: false
+
+      # A string array of values which specifies the host IPv4/IPv6 addresses for NodePort. Values can be valid IP blocks.
```

Comment:
I feel it should be: "An array of strings which specify...". Probably we should change other places too.

Reply:
Done